jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java

```diff
@@ -117,6 +121,13 @@ static String buildToDockerDaemonAndRun(
     return new Command("docker", "run", "--rm", imageReference).run();
   }
 
+  private static String configureInsecureRegistryProperty(String imageReference) {
+    return "-D"
+        + PropertyNames.ALLOW_INSECURE_REGISTRIES
```

Comment:
Maybe `"-Djib.allowInsecure...="`, just for consistency with others? And I believe `... + imageReference.startsWith("localhost")` will do `toString()` which will be either `true` or `false`.